commons/src/main/java/org/springboardfoundation/usersservice/dto/UserDto.java

```diff
@@ -68,11 +68,11 @@ public void setOccupation(String occupation) {
         this.occupation = occupation;
     }
 
-    public int getMobileNumber() {
+    public String getMobileNumber() {
```

Comment:
From this class please remove all the `get` and `set` methods.
On top of the method add annotation `@Data`